Gameheads-SAP-2024/Assets/Scenes/Main Menu.unity

```diff
@@ -172,7 +172,7 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 76c392e42b5098c458856cdf6ecaaaa1, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  m_FirstSelected: {fileID: 722620112}
+  m_FirstSelected: {fileID: 0}
```

Comment:
Change first selected field in the event manager back to a button on the main menu